picoquic/quicctx.c

```diff
@@ -504,16 +501,16 @@ picoquic_cnx_t * picoquic_create_cnx(picoquic_quic_t * quic,
             sizeof(struct sockaddr_in) : sizeof(struct sockaddr_in6));
         memcpy(&cnx->peer_addr, addr, cnx->peer_addr_len);
 
-		picoquic_init_transport_parameters(&cnx->local_parameters, quic->flags&picoquic_context_server);
+        picoquic_init_transport_parameters(&cnx->local_parameters, !cnx->client_mode);
         /* Special provision for test -- create a deliberate transport parameters error */
```

Comment:
If we are going to have a client mode flag, then maybe we should update the init transport parameter implementation. In fact, we may be able to just use a single "connection context" parameter for that function. There are probably other examples like that. Tracking that in issue #64.